score-lang-runtime/src/main/java/org/openscore/lang/runtime/bindings/LoopsBinding.java

```diff
@@ -86,10 +88,17 @@ private LoopCondition createForLoopCondition(ForLoopStatement forLoopStatement,
             throw new RuntimeException("Error evaluating for loop expression in task '" + nodeName + "', error is: \n" + t.getMessage(), t);
         }
 
-        if ((forLoopStatement instanceof MapForLoopStatement) && !(evalResult instanceof PyList)) {
-            throw new RuntimeException(
-                    "Invalid expression for iterating maps: " + collectionExpression
-                    + "\nValid format is: map.items()");
+        if (forLoopStatement instanceof MapForLoopStatement) {
+            if (evalResult instanceof Map) {
+                List<Map.Entry<Serializable, Serializable>> entriesAsSerializable = Lists.newArrayList();
```

Comment:
Depends, do we want to admit non-String keys for maps in slang files: e.g. `k, v in { 1: "abc", 2: "efg" }` ? 

```
k=1
v="abc"
----------
k=2
v="egf"
```